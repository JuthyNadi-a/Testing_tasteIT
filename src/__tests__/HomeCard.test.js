import React from "react";
import renderer from "react-test-renderer";

import HomeCard from '../components/HomeCard';

it('renders when there is card details', () => {
    const tree = renderer.create(<HomeCard details={'testing'}/>).toJSON();
    expect(tree).toMatchSnapshot();
})