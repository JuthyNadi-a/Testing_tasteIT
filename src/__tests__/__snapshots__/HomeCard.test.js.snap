// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is card details 1`] = `
<div
  className="card"
>
  <p
    className="heading"
  />
  <p>
    testing
  </p>
  <a
    className="link"
  />
</div>
`;
